src/main/java/com/shopify/ShopifySdk.java

```diff
@@ -1217,6 +1321,10 @@ public String getAccessToken() {
 		return mapPagedResponse(shopifyCustomerSavedSearchesRoot.getCustomerSavedSearches(), response);
 	}
 
+//	private int getCustomerSavedSearchesCount(){
```

Comment:
Need to test Customer Saved Search Count end point. To verify if it follows the use-case by taking appropriate request parameters